pkg/api-server/resource_endpoints.go

```diff
@@ -174,6 +174,13 @@ func (r *resourceEndpoints) addFindEndpoint(ws *restful.WebService, pathPrefix s
 			Param(ws.PathParameter("outbound_kri", "KRI of a outbound").DataType("string")).
 			Returns(200, "OK", nil).
 			Returns(404, "Not found", nil))
+		ws.Route(ws.GET(pathPrefix+"/{name}/outbounds/{outbound_kri}/_routes/{route_kri}/_policies").To(r.getPoliciesConf(ordered.Policies, matchedPoliciesToRouteConfig)).
```

Comment:
we could simplify this endpoint as we don't actually need outbound kri for this. We could do:
```
"/{name}/routes/{route_kri}/_policies"
```